### DISCUSSION TITLE
How to reuse/recycle Sozi presentations?

### DISCUSSION THREAD

**Discussion by Sprebecca** [senshu/Sozi#623] [Q&A]
I prepared a presentation with Inkscape and Sozi and I would like to use at least the Sozi file (and also the Inkcape file) for a similar presentation in a different context. However, I am quite a unexperienced with technical issues so I don't really know how to recycle/reuse my files.

I tried to change the name of the Inkscape file, but then of course the Sozi/ json-file with the frames of the old presentations don't work anymore.
So how can I copy/reuse my old Inkscape/Sozi files with another name without losing my sozi frames? What is the process?

**Comment by aumouvantsillage** [comment]
The *source* files of a presentation are your original SVG and the JSON file created by Sozi.

If you have a presentation with files `X.svg` and `X.sozi.json`, you can copy and rename them to `Y.svg` and `Y.sozi.json`. Then open `Y.svg` in Sozi.
